requirements.txt

```diff
@@ -15,7 +15,7 @@ aiida-optimize~=0.4
 aiida-qeq~=1.0.0a1
 aiida-quantumespresso~=3.4
 aiida-raspa~=1.2
-aiida-siesta~=1.1
+#aiida-siesta~=1.1
```

Comment:
If we comment it out, then we should also provide a reason, otherwise just remove it.

Reply:
removed.